layers/install/mos-install

```diff
@@ -29,7 +29,7 @@ EOF
 cat > /etc/systemd/system/zot.service << EOF
 [Unit]
 Description=OCI Distribution Registry
-After=network.target local-fs.target
+After=network-online.target local-fs.target
```

Comment:
Note that your commit subject mentions network-online.service, but you implemented the correct unit, network-online.target

Reply:
THanks, updated.